style.css

```diff
@@ -0,0 +1,521 @@
+body{
+    width: 100%;
+    max-width: 1920px;
+    margin: 0 auto;
+    font-family: 'Nanum Gothic', sans-serif;
+    background-color: #ebebeb;
+}
+
+nav{
```

Comment:
I'd highly recommend you to use classes instead of tags in case of browser performance improvement and styles compatibility issues with other libs